src/components/Container/index.tsx

```diff
@@ -73,21 +75,35 @@ function Container(props: Props) {
         footerQuickActions,
         horizontallyCompactContent,
         headerElementProps,
-        elementRef,
+        elementRef: elementRefFromProps,
         headingContainerClassName,
         inlineHeadingDescription,
+        autoFocus,
     } = props;
 
+    const internalRef = React.useRef<HTMLDivElement>(null);
+    const elementRef = elementRefFromProps ?? internalRef;
+
+    React.useEffect(() => {
+        if (autoFocus && elementRef.current) {
+            elementRef.current.scrollIntoView({
+                behavior: 'smooth',
+                block: 'center',
+            });
+        }
+    }, [autoFocus, elementRef]);
```

Comment:
```suggestion
    }, [autoFocus]);
```
I guess we should add this elementRef as ignore?

Reply:
Cannot be ignored